text/0018-bee-bundle/0018-bee-bundle.md

```diff
@@ -0,0 +1,281 @@
++ Feature name: `bee-bundle`
++ Start date: 2019-10-02
++ RFC PR: [iotaledger/bee-rfcs#18](https://github.com/iotaledger/bee-rfcs/pull/18)
++ Bee issue: [iotaledger/bee#63](https://github.com/iotaledger/bee/issues/63)
+
+# Summary
+
+This RFC is based on [`transaction-module`]().
+<!-- TODO FIx link to RFC#3 -->
+
+The smallest communication unit in the IOTA protocol is the transaction. Everything, including payment settlements and/or plain data, is propagated through the IOTA network in transactions.
+
+A transaction is `2673` trytes and the part available to the user is `2187` trytes. This part holds a signature in case of a payment settlement and plain data otherwise. Since it has a limited size, a user often needs more than one transaction to fulfil their operation, for example signatures with security level `2` or `3` don't fit in a single transaction and user-provided data may exceed the allowance so they need to be fragmented across multiple transactions. Moreover, a value transaction doesn't make sense on its own because it would change the total amount of the ledger so it has to be paired with other complementary transactions that together will balance the total value to zero.
```

Comment:
Will do it for the sake of consistency. But when talking about user-facing sizes we usually prefer trytes over trits because of the nice readable mapping. And trits are usually used in crypto contexts.

Reply:
I would recommend stay in trytes and maybe just mention it's 3 trits 1 trytes. Many documentations and other libraries across IOTA ecosystem are still in form of trytes when it comes to build transaction and bundles.

Reply:
@thibault-martinez okey understand what you mean. however, in general, we shouldn't use the term `value transaction` for transfering value, and maybe make this clear in the RFC.